ProcessLib/LiquidFlow/LiquidFlowLocalAssembler.h

```diff
@@ -39,23 +42,27 @@ class LiquidFlowLocalAssemblerInterface
       public NumLib::ExtrapolatableElement
 {
 public:
-    virtual std::vector<double> const& getIntPtDarcyVelocityX(
-        const double /*t*/,
-        GlobalVector const& /*current_solution*/,
-        NumLib::LocalToGlobalIndexMap const& /*dof_table*/,
-        std::vector<double>& /*cache*/) const = 0;
+    LiquidFlowLocalAssemblerInterface(
+        StaggeredCouplingTerm* const coupling_term)
+        : _coupling_term(coupling_term)
+    {
+    }
 
-    virtual std::vector<double> const& getIntPtDarcyVelocityY(
-        const double /*t*/,
-        GlobalVector const& /*current_solution*/,
-        NumLib::LocalToGlobalIndexMap const& /*dof_table*/,
-        std::vector<double>& /*cache*/) const = 0;
+    void setStaggeredCouplingTerm(std::size_t const /*mesh_item_id*/,
+                                  StaggeredCouplingTerm* const coupling_term)
+    {
+        _coupling_term = coupling_term;
+    }
 
-    virtual std::vector<double> const& getIntPtDarcyVelocityZ(
+    virtual std::vector<double> const& getIntPtDarcyVelocity(
         const double /*t*/,
         GlobalVector const& /*current_solution*/,
         NumLib::LocalToGlobalIndexMap const& /*dof_table*/,
         std::vector<double>& /*cache*/) const = 0;
+
+protected:
+    /// Pointer that gets from Process class
```

Comment:
Maybe "Pointer that is set from a Process class" would be better....

Reply:
Changed accordingly.